src/student_success_tool/gcsutils.py

```diff
@@ -0,0 +1,71 @@
+"""
+Utility functions that interact with Google Cloud Storage.
+For instance, copying files from Databricks volumes into GCP buckets.
+"""
+
+from google.cloud import storage
+from google.cloud.storage.bucket import Bucket
```

Comment:
I don't see this package listed as a dependency in `pyproject.toml` -- where's it coming from?